README.md

```diff
@@ -1,2 +1,27 @@
 Kurtosis Lambda Starter Pack
 ============================
+This repo contains:
+
+1. Libraries in various languages for create custom implementations of Kurtosis Lambda
```

Comment:
better: "Libraries in various languages for creating a custom Kurtosis Lambda implementation"